Cogs.Dto/CogsDirectoryReader.cs

```diff
@@ -81,7 +81,7 @@ public CogsDtoModel Load(string directory)
             LoadDataTypes(directory, "ItemTypes", model, model.ItemTypes);
 
             // Load all reusable types from the ReusableTypes directory.
-            LoadDataTypes(directory, "ReusableTypes", model, model.ReusableDataTypes);
+            LoadDataTypes(directory, "CompositeTypes", model, model.ReusableDataTypes);
```

Comment:
Will this change break the integration tests without renaming the cogsburger CompositeTypes directory?